modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -377,7 +379,22 @@ public String toModelFilename(String name) {
 
     @Override
     public String toModelImport(String name) {
-        return modelPackage() + "/" + toModelFilename(name);
+        // with modelNamePrefix/modelNameSuffix, name already includes
```

Comment:
The implementation lookgs good and well-documented.
However, is there a reason why this isnt implemented in the ‚toModelFilename()‘ method?

Reply:
Yes, if you implement it in toModelFilename, the file name will become Error.ts, but the import will stay the same:
import { SwgError} from '../model/swgError';

I did run a test to confirm.